tests/unit/test_questions/conftest.py

```diff
@@ -26,3 +29,22 @@ def conditional_1(question_binary, question_numeric):
             question_type=Question.QuestionType.NUMERIC, title="If No"
         ),
     )
+
+
+@pytest.fixture()
+def question_binary_with_forecast_user_1(user1):
+    post = factory_post()
+    question = create_question(
+        post=post,
+        question_type=Question.QuestionType.BINARY,
+        open_time=datetime(2000, 1, 1, tzinfo=dt_timezone.utc),
+        scheduled_close_time=datetime(3000, 1, 1, tzinfo=dt_timezone.utc),
+    )
+    post.question = question
+    post.save()
```

Comment:
A small nit: `create_question` populates Question object which does not have `post` field, so `post=post` makes not sense. You can move post creation here: `post = factory_post(question=question)`

Reply:
Make sense. Ok!